sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -58,6 +58,15 @@ if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
     DEPENDEES install
   )
 
+  if (WIN32)
+    # Copy DLL into binary directory
+    ExternalProject_Add_Step(level-zero-loader llvmbininstall
+      COMMAND ${CMAKE_COMMAND} -E copy_directory <INSTALL_DIR>/bin/ ${LLVM_BINARY_DIR}/bin
+      COMMENT "Installing level-zero-loader into the LLVM binary directory"
+      DEPENDEES install
+    )
+  endif()
+
```

Comment:
why is this change necessary in this PR and in general?

Reply:
On Windows the .dll for the level-zero loader was in /bin but wasn't copied to the install folder. This would cause testing to fail as it insisted on loading it. Arguably it could be a separate PR.

Reply:
Let's keep it separate. Today Level-zero testing is working, I assume, so I don't understand this fix at least. Aren't DLL searched in %PATH% on Windows?

Reply:
> Aren't DLL searched in %PATH% on Windows?

I believe so, but the problem I encountered while testing was that the .dll for the level-zero loader wasn't copied into anywhere the tests could find it with the environment they set up. I tried rebuilding without these changes and it seems to have been resolved separately.

@AidanBeltonS These changes to sycl/plugins/level_zero/CMakeLists.txt can be removed.